apps/web-mzima-client/src/app/feed/feed.component.html

```diff
@@ -272,5 +272,10 @@ <h3 class="menu-title">{{ 'app.mark_as' | translate }}</h3>
 </ng-template>
 
 <ng-template #postsEmpty>
-  <span class="posts-empty">{{ 'post.no_posts_yet' | translate }}</span>
+  <span class="posts-empty" *ngIf="filters.form[0] === 'none'; else noPosts">{{
+    'post.no_posts_yet' | translate
+  }}</span>
+  <ng-template #noPosts>
+    <span class="posts-empty">There are no posts in the survey!</span>
```

Comment:
Add the string to the en.json-file instead and map it in a similar way as is already done above, on row 276

Reply:
alright @Angamanga But it should be done for english only or for all the available languages?